sherpa-onnx/csrc/hypothesis.h

```diff
@@ -31,6 +34,14 @@ struct Hypothesis {
   // LM log prob if any.
   double lm_log_prob = 0;
 
+  int32_t cur_scored_pos = 0;      // cur scored tokens by RNN LM
+  CopyableOrtValue rnnlm_state_h;  // states for RNN LM
```

Comment:
Could you replace it with
```
std::vector<CopyableOrtValue> lm_states;
```
?

We may support transformer LM later.

Also, making it a vector also simplifies the code in `online-lm.cc`.